packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -188,7 +188,12 @@ export async function getDefinitions(req: AuthRequest, res: Response) {
       };
     }),
     dimensions,
-    segments,
+    segments: segments.map((segment) => {
+      return {
+        ...segment,
+        type: segment.type || "SQL", // if no type, its an old sql type segment
+      };
+    }),
```

Comment:
This logic belongs in a migration within the Segment model. That way, everywhere that segments are used, you can rely on this property existing.